README.md

```diff
@@ -50,42 +50,7 @@ pip install astro-sdk-python[amazon,google,snowflake,postgres]
 
 1. Copy the following DAG into a file named `calculate_popular_movies.py` and add it to the `dags` directory of your Airflow project:
 
-    ```Python
-    from datetime import datetime
-    from airflow import DAG
-    from astro import sql as aql
-    from astro.files import File
-    from astro.sql.table import Table
-
-    @aql.transform()
-    def top_five_animations(input_table: Table):
-        return """
-            SELECT Title, Rating
-            FROM {{input_table}}
-            WHERE Genre1=='Animation'
-            ORDER BY Rating desc
-            LIMIT 5;
-        """
-
-    with DAG(
-        "calculate_popular_movies",
-        schedule_interval=None,
-        start_date=datetime(2000, 1, 1),
-        catchup=False,
-    ) as dag:
-        imdb_movies = aql.load_file(
-            File("https://raw.githubusercontent.com/astronomer/astro-sdk/main/tests/data/imdb.csv"),
-            output_table=Table(
-                name="imdb_movies", conn_id="sqlite_default"
-            ),
-        )
-        top_five_animations(
-            input_table=imdb_movies,
-            output_table=Table(
-                name="top_animation"
-            ),
-        )
-    ```
+    curl -O https://raw.githubusercontent.com/astronomer/astro-sdk/main/example_dags/calculate_popular_movies.py
```

Comment:
@utkarsharma2 why did you favour this approach instead of adding the [code snipped with permalink](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet#linking-to-code)? Is it because the permalink would have the version of the code hard-coded?

Reply:
I would use the permalink, otherwise we could end up having a 404 (completely missing) in our README.

Reply:
@tatiana initially permalinks was my approach - https://github.com/astronomer/astro-sdk/pull/681/commits/21f7bd8edaa7592098447969a7d8d84f5e849778#r949124948 

But I guess @kaxil felt the alternative would be better. I personally incline towards permalinks approach.

Reply:
I felt that mainly because users would otherwise have to copy/paste it and because of scrolling involved with code-snippet that might result in issues.

However, if all of you think snippet + permalink is a better approach then go for it.

Reply:
@utkarsharma2 What if we kept both of them? The code snipped & the curl command - so we would meet the requirement from the doc team while also simplifying for those who may want to download it straightway - without having to have the code hard-coded in the README?

Reply:
@tatiana I think we can keep both if we are able to verify the curl link is working. Just a little worried if move the file somewhere else and the links are broken.

Reply:
@tatiana Never mind - CI job takes cares of that - gaurav-nelson/github-action-markdown-link-check@v1

Reply:
Updated: 

![Screenshot 2022-09-07 at 2 34 48 AM](https://user-images.githubusercontent.com/13021213/188737974-d49af6ec-3b36-45c3-850d-10559b70dd5c.png)

Reply:
> Never mind - CI job takes care of that - gaurav-nelson/github-action-markdown-link-check@v1. I'm going ahead with adding both.

Right. That's great! +1 for using `main` version.